src/main/java/io/github/christophermanahan/carnitas/Request.java

```diff
@@ -0,0 +1,6 @@
+package io.github.christophermanahan.carnitas;
+
+public interface Request {
```

Comment:
I'm not completely sure what `Request` and `RequestBuilder` have interfaces

Reply:
The interfaces have been removed.